### DISCUSSION TITLE
How to change break points for Mobile Menu

### DISCUSSION THREAD

**Discussion by southpaw3229** [templaza/astroid-framework#197] [Q&A] [answered]
I believe I can use some custom css but wanted to know if the framework has a section to enter a certain break point for when the mobile menu appears?

I did apply some custom code to reduce the right and left padding left of logo and right of main nav to help reduce my menu items from creating 2 rows, but wanted to force the mobile menu higher than 990.

Thanks and love the framework.  Keep it going.

**Comment by sonvnn** [comment, marked as answer]
Hi @southpaw3229 

You can edit scss code

- Create a variable_overrides.scss file save it to media/templates/site/{your-template}/variable_overrides.scss
- Add this code below to variable_overrides.scss.

```
$grid-breakpoints: (
  xs: 0,
  sm: 576px,
  md: 768px,
  lg: 992px,
  xl: 1200px,
  xxl: 1400px
) !default;
```

Best Regards,
Sonny
